### PR TITLE
docs:update readme.md

### LINKED ISSUE #711: docs:update readme.md
- [ ] I have searched the [issues](https://github.com/alibaba/fescar/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
docs:update readme.md
logo、fescar->seata、 url

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- OS :
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
### Ⅰ. Describe what this PR did

docs:update readme.md

### Ⅱ. Does this pull request fix one issue?
fix #711 docs:update readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ A typical lifecycle of Fescar managed distributed transaction:
 
 ![Typical Process](https://cdn.nlark.com/lark/0/2018/png/18862/1545296917881-26fabeb9-71fa-4f3e-8a7a-fc317d3389f4.png) 
 
-For more details about principle and design, please go to [Fescar wiki page](https://github.com/alibaba/fescar/wiki). 
+For more details about principle and design, please go to [Seata wiki page](https://github.com/seata/seata/wiki). 
 
 ### History
```
Comment (zhangthen) [inline README.md]: History  是否需要加一个Seata ?

File: README.md [hunk 2]
```diff
@@ -90,15 +90,15 @@ For more details about principle and design, please go to [Fescar wiki page](htt
 ```
 ## Quick Start
 
-[Quick Start](https://github.com/alibaba/fescar/wiki/Quick-Start)
+[Quick Start](https://github.com/seata/seata/wiki/Quick-Start)
 
 ## Documentation
 
-You can view the full documentation from the wiki: [Fescar wiki page](https://github.com/alibaba/fescar/wiki).
+You can view the full documentation from the wiki: [Fescar wiki page](https://github.com/seata/seata/wiki).
```
Comment (xingfudeshi) [inline README.md]: Seata wiki page

File: README.md [hunk 3]
```diff
@@ -118,20 +118,20 @@ Contributors are welcomed to join the FEATS project. Please check [CONTRIBUTING]
...
-## Fescar ecosystem
+## Seata ecosystem
 
-* [Fescar Ecosystem Entry](https://github.com/fescar-group) - A GitHub group `fescar-group` to gather all Fescar relevant projects not appropriate in [alibaba](https://github.com/alibaba) group yet
-* [Fescar Samples](https://github.com/fescar-group/fescar-samples) - Samples for Fescar
-* [Fescar Docker](https://github.com/fescar-group/fescar-docker) - Fescar integration with docker
-* [Fescar K8s](https://github.com/fescar-group/fescar-k8s) - Fescar integration with k8s
-* [Awesome Fescar](https://github.com/fescar-group/awesome-fescar) - Description of Fescar related projects 
-* [Fescar Website](https://github.com/fescar-group/fescar.io) - Fescar official website （***In the process of design***）
+* [Seata Ecosystem Entry](https://github.com/fescar-group) - A GitHub group `fescar-group` to gather all Fescar relevant projects not appropriate in [alibaba](https://github.com/alibaba) group yet
```
Comment (xingfudeshi) [inline README.md]: A GitHub group `seata-group`,....all Seata relevant

File: README.md [hunk 4]
```diff
@@ -118,20 +118,20 @@ Contributors are welcomed to join the FEATS project. Please check [CONTRIBUTING]
...
+* [Seata Samples](https://github.com/fescar-group/fescar-samples) - Samples for Fescar
```
Comment (xingfudeshi) [inline README.md]: Samples for Seata

File: README.md [hunk 5]
```diff
@@ -118,20 +118,20 @@ Contributors are welcomed to join the FEATS project. Please check [CONTRIBUTING]
...
+* [Seata Docker](https://github.com/fescar-group/fescar-docker) - Seata integration with docker
+* [Seata K8s](https://github.com/fescar-group/fescar-k8s) - Seata integration with k8s
+* [Awesome Seata](https://github.com/fescar-group/awesome-fescar) - Description of Fescar related projects
```
Comment (xingfudeshi) [inline README.md]: Description of Seata

File: README.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
-<img src="https://github.com/fescar-group/fescar-samples/blob/master/doc/img/fescar.png"  height="100" width="426">
+<img src="https://github.com/fescar-group/fescar-samples/blob/master/doc/img/seata.png"  height="100" width="426">
 
-# Fescar: Fast & Easy Commit And Rollback
+# Seata: Simple Extensible Autonomous Transaction Architecture
 
 [![Build Status](https://travis-ci.org/alibaba/fescar.svg?branch=develop)](https://travis-ci.org/alibaba/fescar)
```
Comment (xingfudeshi) [inline README.md]: can these svg resource be replaced?

Comment (slievrly) [reply README.md]: Temporarily not, not yet established

File: README.md (hunk 1 above)
Comment (slievrly) [reply README.md]: Wait for the operation to give this information, this version temporarily replaces only the name

File: README.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 [![codecov](https://codecov.io/gh/alibaba/fescar/branch/develop/graph/badge.svg)](https://codecov.io/gh/alibaba/fescar)
 ![license](https://img.shields.io/github/license/alibaba/fescar.svg)
 ![maven](https://img.shields.io/maven-central/v/com.alibaba.fescar/fescar-all.svg)
```
Comment (lovepoem) [inline README.md]: Please update the budges from alibaba/fescar to seata/seata

File: README.md (hunk 3 above)
Comment (lovepoem) [reply README.md]: When to transfer the projects under https://github.com/fescar-group/awesome-fescar

**Pull request merged**